*** Android.mk	2016-01-03 07:15:57.000000000 +0100
--- Android.mk.new	2016-01-29 20:28:30.655967056 +0100
***************
*** 11,22 ****
  
  # Enable this if you want to support loading MOD music via mikmod
  # The library path should be a relative path to this directory.
! SUPPORT_MOD_MIKMOD ?= true
  MIKMOD_LIBRARY_PATH := external/libmikmod-3.1.12
  
  # Enable this if you want to support loading MP3 music via SMPEG
  # The library path should be a relative path to this directory.
! SUPPORT_MP3_SMPEG ?= true
  SMPEG_LIBRARY_PATH := external/smpeg2-2.0.0
  
  # Enable this if you want to support loading OGG Vorbis music via Tremor
--- 11,22 ----
  
  # Enable this if you want to support loading MOD music via mikmod
  # The library path should be a relative path to this directory.
! SUPPORT_MOD_MIKMOD ?= false
  MIKMOD_LIBRARY_PATH := external/libmikmod-3.1.12
  
  # Enable this if you want to support loading MP3 music via SMPEG
  # The library path should be a relative path to this directory.
! SUPPORT_MP3_SMPEG ?= false
  SMPEG_LIBRARY_PATH := external/smpeg2-2.0.0
  
  # Enable this if you want to support loading OGG Vorbis music via Tremor
***************
*** 27,36 ****
  
  
  # Enable this if you want to support TiMidity
! SUPPORT_TIMIDITY ?= true
  
! LOCAL_C_INCLUDES := $(LOCAL_PATH) 
! LOCAL_CFLAGS := -DWAV_MUSIC 
  
  LOCAL_SRC_FILES := $(notdir $(filter-out %/playmus.c %/playwave.c, $(wildcard $(LOCAL_PATH)/*.c))) \
  
--- 27,36 ----
  
  
  # Enable this if you want to support TiMidity
! SUPPORT_TIMIDITY ?= false
  
! LOCAL_C_INCLUDES := $(LOCAL_PATH)/../../SDL2-2.0.4/include
! LOCAL_CFLAGS := -DWAV_MUSIC
  
  LOCAL_SRC_FILES := $(notdir $(filter-out %/playmus.c %/playwave.c, $(wildcard $(LOCAL_PATH)/*.c))) \
  
***************
*** 128,131 ****
  
  LOCAL_ARM_MODE := arm
  
! include $(BUILD_SHARED_LIBRARY)
--- 128,131 ----
  
  LOCAL_ARM_MODE := arm
  
! include $(BUILD_STATIC_LIBRARY)
