*** CMakeLists.txt	2016-01-02 20:56:31.000000000 +0100
--- SDL2.CMakeLists.txt	2016-01-26 21:57:39.154964203 +0100
***************
*** 42,47 ****
--- 42,49 ----
  set(LT_RELEASE "${SDL_MAJOR_VERSION}.${SDL_MINOR_VERSION}")
  set(LT_VERSION "${LT_MAJOR}.${LT_AGE}.${LT_REVISION}")
  
+ set (CMAKE_POSITION_INDEPENDENT_CODE TRUE)
+ 
  message(STATUS "${LT_VERSION} :: ${LT_AGE} :: ${LT_REVISION} :: ${LT_CURRENT} :: ${LT_RELEASE}")
  
  # General settings & flags
***************
*** 183,190 ****
  set(SDL_LIBS "-lSDL2")
  set(SDL_CFLAGS "")
  
! # Emscripten toolchain has a nonempty default value for this, and the checks 
! # in this file need to change that, so remember the original value, and 
  # restore back to that afterwards. For check_function_exists() to work in
  # Emscripten, this value must be at its default value.
  set(ORIG_CMAKE_REQUIRED_FLAGS ${CMAKE_REQUIRED_FLAGS})
--- 185,192 ----
  set(SDL_LIBS "-lSDL2")
  set(SDL_CFLAGS "")
  
! # Emscripten toolchain has a nonempty default value for this, and the checks
! # in this file need to change that, so remember the original value, and
  # restore back to that afterwards. For check_function_exists() to work in
  # Emscripten, this value must be at its default value.
  set(ORIG_CMAKE_REQUIRED_FLAGS ${CMAKE_REQUIRED_FLAGS})
***************
*** 212,218 ****
  set(OPT_DEF_ASM TRUE)
  if(EMSCRIPTEN)
    # Set up default values for the currently supported set of subsystems:
!   # Emscripten/Javascript does not have assembly support, a dynamic library 
    # loading architecture, low-level CPU inspection or multithreading.
    set(OPT_DEF_ASM FALSE)
    set(SDL_SHARED_ENABLED_BY_DEFAULT OFF)
--- 214,220 ----
  set(OPT_DEF_ASM TRUE)
  if(EMSCRIPTEN)
    # Set up default values for the currently supported set of subsystems:
!   # Emscripten/Javascript does not have assembly support, a dynamic library
    # loading architecture, low-level CPU inspection or multithreading.
    set(OPT_DEF_ASM FALSE)
    set(SDL_SHARED_ENABLED_BY_DEFAULT OFF)
***************
*** 836,842 ****
          #include <linux/kd.h>
          #include <linux/keyboard.h>
  
!         int main(int argc, char **argv) 
          {
              struct kbentry kbe;
              kbe.kb_table = KG_CTRL;
--- 838,844 ----
          #include <linux/kd.h>
          #include <linux/keyboard.h>
  
!         int main(int argc, char **argv)
          {
              struct kbentry kbe;
              kbe.kb_table = KG_CTRL;
